### ISSUE TITLE
JAWS fails to read last cell in a nested table, MS Edge on Win10

### ISSUE THREAD

**Issue by skingsland** [FreedomScientific/VFO-standards-support#163]
### Summary

1. Using MS Edge browser on Windows 10, go to https://skingsland.github.io/JAWS_BUG/index.html
2. Have JAWS read the page

### Expected result

All the text on the page is read aloud.

### Actual result

JAWS doesn't read the text in the last cell in the table. Instead it says "blank".

### Example

Here is the HTML from the page I linked above:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
</head>
<body>
    <table>
        <tr>
            <td>
                <table>
                    <tr>
                        <td>Cell 1</td>
                        <td>This cell is NOT read by JAWS!</td>
                    </tr>
                </table>
            </td>
        </tr>

        <!-- this row is required to cause the bug, but the TD must be EMPTY -->
        <tr>
            <td></td>
        </tr>
    </table>

    <!-- this table is required to cause the bug -->
    <table>
        <tr>
            <td>second table</td>
        </tr>
    </table>

</body>
</html>
```

### Additional Information

The presence of _both_ the empty row _and_ the second table seems to cause JAWS to think the second cell in the main table is blank.

#### JAWS version and build number
2019.1812.49 ILM Professional Edition

 
#### Operating System and version
Windows 10 Enterprise, build 17134.rs4_release.180410-1804

#### Browser and version:
Microsoft Edge 42.17134.1.0

**Comment by scottaohara** [comment]
Apologies, but since Edge is moving to using the chrome rendering engine we are no longer putting time into current Edge related bugs.

Closing this for now, but we can re-open if the issue still exists once the stable public release version of Edge is available.
